tests/PgSqlServerTest/PgSqlServerTest.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+# vim: set ai et ts=4 sw=4:
+
+import os
+import random
+import subprocess
+import requests
+import logging
+import time
+import socket
+import sys
+import psycopg2
+
+START = os.environ.get('HURMADB_PORT') is None
+PORT = int(os.getenv('HURMADB_PORT', 5432))
+con = None
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+
+def hurmadb_start(port):
+    """
+    Start HurmaDB on given port, return an object that corresponds to the created process.
+    """
+    if START:
+        return subprocess.Popen(['../../hurmadb', str(port)])
+    else:
+        return None
+
+def hurmadb_stop(pobj):
+    """
+    Stop HurmaDB process created by hurmadb_start()
+    """
+    if con:
+        con.close()
+
+    if START:
+        pobj.wait()
+
+class TestBasic:
```

Comment:
This test doesn't cover current class functionality since it doesn't execute any commands.